docs/building-on-etherlink/bridging-wrapped-assets.mdx

```diff
@@ -81,221 +86,209 @@ The transfer button can show these errors:
-```
-
-- [WETH](https://basescan.org/address/0x4200000000000000000000000000000000000006) - Wrapped Ether
-```
-0x4200000000000000000000000000000000000006
-```
-
-:::
-
-#### Optimism
-
-:::note Token Addresses
-
-- [USDC](https://optimistic.etherscan.io/address/0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85) - USD Coin
-```
-0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85
-```
-
-- [USDT](https://optimistic.etherscan.io/address/0x94b008aA00579c1307B0EF2c499aD98a8ce58e58) - Tether USD
-```
-0x94b008aA00579c1307B0EF2c499aD98a8ce58e58
-```
-
-- [WETH](https://optimistic.etherscan.io/address/0x4200000000000000000000000000000000000006) - Wrapped Ether
-```
-0x4200000000000000000000000000000000000006
-```
-
-:::
-
-#### BNB Chain
-
-:::note Token Addresses
-
-- [USDC](https://testnet.bscscan.com/address/0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d) - USD Coin
-```
-0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d
-```
-
-- [USDT](https://testnet.bscscan.com/address/0x55d398326f99059fF775485246999027B3197955) - Tether USD
-```
-0x55d398326f99059fF775485246999027B3197955
-```
-
-- [WBNB](https://testnet.bscscan.com/address/0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c) - Wrapped BNB
-```
-0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c
-```
-
-:::
-
-#### Avalanche C-Chain
-
-:::note Token Addresses
-
-- [USDC](https://avascan.info/blockchain/c/address/0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E) - USD Coin
-```
-0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E
-```
-
-- [USDT](https://avascan.info/blockchain/c/address/0x9702230A8Ea53601f5cD2dc00fDBc13d4dF4A8c7) - Tether USD
-```
-0x9702230A8Ea53601f5cD2dc00fDBc13d4dF4A8c7
-```
-
-- [WAVAX](https://avascan.info/blockchain/c/address/0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7) - Wrapped AVAX
-```
-0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7
-```
-
-:::
-
-## Useful Links
-
-More Information about the LayerZero protocol is available in their [official documentation](https://docs.layerzero.network/).
-
-The Audit Reports of the LayerZero protocol are available in this [LayerZero GitHub Repository](https://github.com/LayerZero-Labs/LayerZero/tree/main/audit).
-
-To review the Open Source code of the Wrapped Asset Bridge, visit the [LayerZero Github Repository](https://github.com/LayerZero-Labs/wrapped-asset-bridge).
-
-
-{/* ## How bridging wrapped assets works
-
-The Etherlink wrapped asset bridge uses a [wrapped asset bridge](https://github.com/LayerZero-Labs/wrapped-asset-bridge) created by [LayerZero](https://layerzero.network/).
-It works by maintaining pools on each side of the bridge.
-
-When you bridge tokens, it locks the tokens on the source network, sends a message over LayerZero's software to the target network, and unlocks wrapped tokens on the target network.
-
-For more information, see the repository for the bridge: https://github.com/LayerZero-Labs/wrapped-asset-bridge. -->
-*/}
\ No newline at end of file
+## Token addresses
+
+For the addresses of wrapped tokens on Etherlink, see [Tokens](./tokens).
```

Comment:
The only thing is not all the tokens on the token page on Etherlink will be bridgeable on this bridge, for example, WXTZ isn't something included.

Reply:
Added "Not all Etherlink tokens can be bridged in and out of Etherlink."